spdmlib/src/crypto/spdm_ring/aead_impl.rs

```diff
@@ -2,9 +2,9 @@
 //
 // SPDX-License-Identifier: BSD-2-Clause-Patent
 
+use crate::crypto::bytes_mut_scrubbed::BytesMutStrubbed;
```

Comment:
this is for ring only.
Do we need this for mbedtls binding?

Reply:
> this is for ring only. Do we need this for mbedtls binding?

The implementation of mbedtls binding doesn't use BytesMut at all. Use the raw u8 slice.
How about let's create a new issue to track this?

Reply:
yes, let's create another issue

Reply:
> yes, let's create another issue

created here https://github.com/jyao1/rust-spdm/issues/605